project/OmnidocBuild.scala

```diff
@@ -335,7 +335,7 @@ object OmnidocBuild {
     }
   }
 
-  def sourceUrlMarker(path: String): String = s"http://%SOURCE;${path}%"
+  def sourceUrlMarker(path: String): String = s"http://_SOURCE;${path}_"
```

Comment:
Scala 2.13.11 switched from `java.net.URL` to `java.net.URI` (https://github.com/scala/scala/commit/258a13bab73a70858549e6cbc1c1ec9757fbcc3f#diff-cf9c3707d670d80fd740a9a4e61b681ae0a2a6f502198553c0083ad3bbc994d6R334), which causes 
```
[error] (Omnidoc / scaladoc) java.net.URISyntaxException: Malformed escape pair at index 7: http://%SOURCE;/org.scalatestplus.play-scalatestplus-play_2.13-6.0.0-M6/org/scalatestplus/play/components/OneAppPerTestWithComponents%
```
To solve this I just replaced the percent sign with an underscore. I checked the generated files and they are identical, so I assume it's ok to to do that. Like the method name says, it's a maker, that will be set and later be replaced again via a regex, so the maker is just used temporary.

Reply:
Note to myself: This marker variable was added in 3a1e08dcdc5f1f3d596e14c766a6fd012d8a01c8